product_docs/docs/pgd/5/rel_notes/pgd_5.2.0_rel_notes.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Release notes for EDB Postgres Distributed version 5.2.0"
+navTitle: "Version 5.2.0"
+---
+
+EDB Postgres Distributed version 5.2.0 is a minor version of EDB Postgres Distributed.
+This version addresses security vulnerabilities in dependencies for PGD Proxy and PGD CLI.
+
+## Highlights of EDB  Postgres Distributed 5.2
+
+*
+
+!!! Important Recommended upgrade
+    We recommend that users of PGD 5.1 upgrade to PGD 5.2.
+
+!!! Note PostgreSQL version compatibility
+This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.8, and later.
+!!!
+
+
+| Component | Version | Type            | Description  |
+|-----------|---------|-----------------|--------------|
+
+| BDR       | 5.2.0   | Feature         | Sub-group Raft |
```

Comment:
Sub-group Raft was added in 5.0 what does this refer to?

Reply:
Removed